pipeline/transformer/transformer_sierra/src/main/scala/weco/pipeline/transformer/sierra/transformers/SierraPhysicalDescription.scala

```diff
@@ -35,13 +30,17 @@ object SierraPhysicalDescription
 
   type Output = Option[String]
 
-  def apply(bibData: SierraBibData) =
-    bibData
-      .subfieldsWithTags(
-        "300" -> "a",
-        "300" -> "b",
-        "300" -> "c",
-        "300" -> "e",
-      )
-      .contentString(" ")
+  def apply(bibData: SierraBibData): Option[String] = {
+    val lines =
+      bibData
+        .varfieldsWithTag("300")
+        .flatMap { vf =>
+          vf.subfieldsWithTags("a", "b", "c", "e").contentString(" ")
+        }
+
+    lines.mkString("<br/>") match {
```

Comment:
Could you give some examples of what content we have the we'll split across multiple lines? I ask as I wonder if this is the correct semantics for it?

Ping @gestchild

Reply:
> 1 videocassette (VHS) (1 min.) : sound, color, PAL.
> 1 DVD (1 min.) : sound, color

is a typical example.

Reply:
Yea, strange that's probably more of a list, which it would be good if we could return that, _but_ that would be a massive change.